integration/spark/shared/build.gradle

```diff
@@ -47,7 +47,7 @@ dependencies {
     compileOnly("org.apache.spark:spark-sql-kafka-0-10_${scalaBinaryVersion}:${sparkVersion}")
     compileOnly("org.apache.spark:spark-sql_${scalaBinaryVersion}:${sparkVersion}")
 
-    compileOnly("com.databricks:databricks-dbutils-scala_${scalaBinaryVersion}:${databricksVersion}") {
+    implementation("com.databricks:databricks-dbutils-scala_${scalaBinaryVersion}:${databricksVersion}") {
```

Comment:
ohh, I thought we don't want to add this dependency. We just want to make use its classes if it is on the classpath. Otherwise we end up accumulating all the dependencies from all the spark runtimes, extensions and libs.

Reply:
I think we should not include any extra packages in the shadow jar, this way we don't end up with a really fat jar 🤣
For this, I suggest that we roll back this imports in the [commit](https://github.com/OpenLineage/OpenLineage/commit/89beaadb9a608244e5375d0f76a124f78c5167d8#diff-5542f4a02d45d03e2fdc704d8bf3aab869f919d72a4785a077c52acf86c3aa6a) since `com.databricks.dbutils_v1` that has `com.databricks.dbutils_v1.DbfsUtils` already exists on the `classpath`.
If we don't do so people will be struggling installing OpenLineage in Databricks.